main.js

```diff
@@ -236,30 +240,34 @@ class openknx extends utils.Adapter {
             //before composite check, date is also composite
             knxVal = new Date(state.val);
         } else if (this.gaList.getDataById(id).native.valuetype == "composite") {
-            try {
-                knxVal = JSON.parse(state.val);
-            } catch (e) {
-                this.log.warn("stateChange: unsupported value format " + state.val + " for " + ga);
-                return;
-            }
+            //input from IOB is either object or string in object notation, type of this conversion is object
+            if (typeof state.val == 'object') {
```

Comment:
we should also allow the knx.0 format
1,0
and convert it to 
{priority: 0, data: 1}